From 514ce4e19b9b3add8a4902be576bc7d85d23d31b Mon Sep 17 00:00:00 2001
From: Julien Vuillaumier <julien.vuillaumier@nxp.com>
Date: Wed, 3 Nov 2021 15:19:04 +0100
Subject: [PATCH] [Test] Fix build issue with GCC 11
 [-Werror=maybe-uninitialized]

Correction for build issue showing up with GCC 11

unittest_common.cc:953:51:
error: 'structure' may be used uninitialized [-Werror=maybe-uninitialized]

Upstream-Status: Pending

Signed-off-by: Julien Vuillaumier <julien.vuillaumier@nxp.com>
---
 tests/common/unittest_common.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/common/unittest_common.cc b/tests/common/unittest_common.cc
index 7270ad10..1d8a3327 100644
--- a/tests/common/unittest_common.cc
+++ b/tests/common/unittest_common.cc
@@ -948,7 +948,7 @@ TEST (commonTensorsConfig, validateInvalidParam2_n)
  */
 TEST (commonTensorsConfig, fromStructreInvalidParam0_n)
 {
-  GstStructure structure;
+  GstStructure structure = { 0 };
 
   EXPECT_FALSE (gst_tensors_config_from_structure (NULL, &structure));
 }
-- 
2.25.1

